src/components/forms/TextInput/TextInput.stories.tsx

```diff
@@ -65,16 +65,6 @@ export const readonly = (): React.ReactElement => (
   <TextInput id="input-readonly" name="input-readonly" type="text" readOnly />
 )
 
-/*
-export const small = (): React.ReactElement => (
-  <TextInput id="input-type-text" name="input-type-text" type="text" inputSize="small" />
-)
-
-export const medium = (): React.ReactElement => (
-  <TextInput id="input-type-text" name="input-type-text" type="text" inputSize="medium" />
-)
-*/
```

Comment:
Deletion of these commented stories corresponds to https://github.com/trussworks/react-uswds/pull/1988#discussion_r851441877